repo-agent/review-ui/review-api/main.go

```diff
@@ -472,6 +485,25 @@ func submitReview(c *gin.Context) {
 		return
 	}
 
+	if draft != agentDraft {
+		// Store feedback for fine-tuning
+		prompt, _, _ := unstructured.NestedString(repoWatch.Object, "spec", "review", "gemini", "prompt")
+		configdir, _, _ := unstructured.NestedString(repoWatch.Object, "spec", "review", "gemini", "configdirRef")
+		repoURL, _, _ := unstructured.NestedString(repoWatch.Object, "spec", "repoURL")
+		owner, _, _ := parseRepoURL(repoURL)
+
+		hfKey := fmt.Sprintf("hf:review:githubuser:%s:repo:%s:pr:%s", owner, repo, prID)
+		if err := rdb.HSet(ctx, hfKey,
```

Comment:
Do we want to have the backend be configurable or have a split backend for redis/memory-items vs postgresql/storage-items?  I am not sure the final architecture but currently the delta feedback (important telemetry) would be wiped on reboots etc.

Is the assumption the redis instance will be long lived and always available?  My only concern would be that effectively we would to export this telemetry and hopefully across N teams and not sure if folks would understand that restarting their instance would wipe all telemetry, etc.

Reply:
this is a great Q. I think this needs to be persistent across installs. Across cluster lifetimes.